packages/components/src/components/form/Radio/Radio.tsx

```diff
@@ -102,6 +102,7 @@ export interface RadioProps {
     onClick: EventHandler<SyntheticEvent>;
     'data-test'?: string;
     children?: ReactNode;
+    className?: string;
```

Comment:
@jvaclavik If I remember correctly, last week you told us something about using this right? There is probably mismatch in radio buttons in ETH staking design and all other designs, could you please check it?

Reply:
Yes, we shouldn't override components from design system, please don't add `className`. 

Related to this: https://github.com/trezor/trezor-suite/pull/11187#discussion_r1496114954

Reply:
https://github.com/everstake/trezor-suite/commit/9d06b0a8da0dc44f40d19f7206507188b57f0e19